src/main/java/org/alliancegenome/mati/interfaces/IdentifierResourceRESTInterface.java

```diff
@@ -9,12 +9,18 @@
 
 import io.quarkus.security.Authenticated;
 
+
+/** Endpoints to mint AGR identifiers and get the last value of a counter  */
 @Path("/identifier")
 @Produces(MediaType.APPLICATION_JSON)
 @Consumes(MediaType.APPLICATION_JSON)
 @Tag(name = "Identifier", description = "identifier Operations")
 public interface IdentifierResourceRESTInterface {
 
+	/** Endpoint for GET
+	 * @param auth_header authorization token
+	 * @param subdomain AGR subdomain
+	 * @return an HTTP response with the last identifier minted for the given subdomain */
```

Comment:
@abecerra does this auto generate swagger docs? Just curious why the addition of javadocs?